nifi-api/src/main/java/org/apache/nifi/processor/util/StandardValidators.java

```diff
@@ -540,9 +540,9 @@ public ValidationResult validate(final String subject, final String input, final
                 }
 
                 try {
+                    // Check that we can parse the value as a URL
                     final String evaluatedInput = context.newPropertyValue(input).evaluateAttributeExpressions().getValue();
-                    final URI uri = UriUtils.create(evaluatedInput);
-                    uri.toURL();
+                    URI.create(evaluatedInput).toURL();
```

Comment:
Lines 544-545 which were removed were meant to fix the bug discovered in NIFI-12513

Reply:
Got it. I saw it was used in InvokeHTTP but didn't think it made sense for validation. But I guess it does.